ui/app/build-configs/directives/pnc-create-build-config-general-form/pnc-create-build-config-general-form.html

```diff
@@ -1,7 +1,7 @@
 <!--
 
     JBoss, Home of Professional Open Source.
-    Copyright 2014 Red Hat, Inc., and individual contributors
+    Copyright 2014-2019 Red Hat, Inc., and individual contributors
```

Comment:
@alexcreasy  I believe the check expects "Copyright 2014 Red Hat, Inc., and individual contributors" here for headers, as it's complaining

Reply:
Thanks, that always gets me! License headers fixed and pushed